### PR TITLE
Fix readme generator rake task

### PR DESCRIPTION
**Pull request by revolter**
This updates the code documentation so the generated README matches the current (manually modified) one.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,15 +76,15 @@ inspector = GhInspector::Inspector.new "fastlane", "fastlane"
 inspector.search_query "Someone set us up the bomb", FastlaneUI.new
 ```
 
-
```
Comment (revolter) [inline README.md]: We don't need 2 empty lines, as one of them gets ignored anyways.

File: README.md [hunk 2]
```diff
@@ -76,15 +76,15 @@ inspector = GhInspector::Inspector.new "fastlane", "fastlane"
...
 Protocol for custom objects:
 
  - `inspector_started_query(query, inspector)` - Called just as the investigation has begun.
+ - `inspector_successfully_recieved_report(report, inspector)` - Deprecated: Please use `inspector_successfully_received_report` instead.
```
Comment (revolter) [inline README.md]: Didn't find a way to make `yard` ignore these, but maybe it's for the better. If an old user comes back to this they'll see that they got deprecated.

File: gh_inspector.gemspec [hunk 3]
```diff
@@ -23,4 +23,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rake', '~> 10.0'
   spec.add_development_dependency 'rspec', '~> 3.0'
   spec.add_development_dependency 'rubocop', '~> 0', '> 0'
+  spec.add_development_dependency 'yard', '~> 0', '> 0'
```
Comment (revolter) [inline gh_inspector.gemspec]: Not sure what `'~> 0', '> 0'` means but thought it allows any version, obviously copied from above.

File: lib/gh_inspector/evidence.rb [hunk 4]
```diff
@@ -2,24 +2,32 @@
 require 'time'
 
 module GhInspector
+  NUMBER_OF_ISSUES_INLINE = 3
```
Comment (revolter) [inline lib/gh_inspector/evidence.rb]: This was making `yard` ignore the documentation below.

File: lib/gh_inspector/evidence.rb [hunk 5]
```diff
@@ -2,24 +2,32 @@
...
+  # or
+  #
+  # ``` ruby
+  # require 'gh_inspector'
+  # inspector = GhInspector::Inspector.new "fastlane", "fastlane"
+  # inspector.search_query "Someone set us up the bomb", FastlaneUI.new
   # ```
   #
-
-  NUMBER_OF_ISSUES_INLINE = 3
```
Comment (revolter) [inline lib/gh_inspector/evidence.rb]: This was making `yard` ignore the documentation below.

File: lib/gh_inspector/evidence.rb [hunk 6]
```diff
@@ -2,24 +2,32 @@
...
 module GhInspector
+  NUMBER_OF_ISSUES_INLINE = 3
+
   # The default user interface for the inspector, its public API should be
   # considered the protocol for other classes wanting to provide a user interface.
   #
   # Your custom objects will be verified at runtime that they conform to the protocol.
   #
-  # You can see the default implmentation at
+  # You can see the default implementation at
```
Comment (revolter) [inline lib/gh_inspector/evidence.rb]: Typo fixed [here](https://github.com/orta/gh_inspector/commit/4b6ed2d567bbc268fc042034c9147c255a1d7702#diff-04c6e90faac2675aa89e2176d2eec7d8R60), alongside the other changes.

**Comment by orta** [comment]
awesome! I missed this

**Comment by revolter** [comment]
@orta, I even forgot about this 😂

**Pull request merged**